README.md

```diff
@@ -467,26 +467,36 @@ pyATS Docker images created using this package features the following directory
 structure:
 
 ```text
-/venv
+/pyats
```

Comment:
why this was changed? I'm worrying about impact where we need to update docs and training materials.